### PR TITLE
Adds Grades

### PR DESCRIPTION
**Pull request by decoles**

### CODE REVIEW

File: grades.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Grades
+- Letter grades (A through F) follow a traditional, descending 10-point scale, where the top 2 points earn a + and bottom 2 points earn a - for each letter grade. There is no extra credit nor grade inflation. Final grades are calculated with the following weights:
```
Comment (kbuffardi) [inline grades.md:2]: Let's remove the bullet-point list for the opening sentence.

File: grades.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Grades
+- Letter grades (A through F) follow a traditional, descending 10-point scale, where the top 2 points earn a + and bottom 2 points earn a - for each letter grade. There is no extra credit nor grade inflation. Final grades are calculated with the following weights:
+- 40% Open Source maintenance portfolio
```
Comment (kbuffardi) [inline grades.md:3]: A list needs an empty line before it

**Review by kbuffardi — CHANGES_REQUESTED** [review]

File: grades.md (hunk 1 above)
Comment (decoles) [reply grades.md]: Done

**Comment by decoles** [comment]
Updated changes

**Review by decoles — COMMENTED** [review]
Added requested fixes

**Comment by kbuffardi** [comment]
Closing since this issue has been resolved

**Pull request closed without merging**